### ISSUE TITLE
Qt issue when running the example code on README.md

### ISSUE THREAD

**Issue by mesllo** [visual-layer/fastdup#48]
I get the below exception when I try to run the following code (the example on the README.md):

```python
import fastdup
fastdup.run(input_dir="/path/to/your/folder", work_dir='out', nearest_neighbors_k=5, turi_param='ccthreshold=0.96')    #main running function.
fastdup.create_duplicates_gallery('out/similarity.csv', save_path='.')     #create a visual gallery of found duplicates
fastdup.create_outliers_gallery('out/outliers.csv',   save_path='.')       #create a visual gallery of anomalies
fastdup.create_components_gallery('out', save_path='.')                    #create visualiaiton of connected components
fastdup.create_stats_gallery('out', save_path='.', metric='blur')          #create visualization of images stastics (for example blur)
fastdup.create_similarity_gallery('out', save_path='.',get_label_func=lambda x: x.split('/')[-2])     #create visualization of top_k similar images assuming data have labels which are in the folder name
fastdup.create_aspect_ratio_gallery('out', save_path='.')                  #create aspect ratio gallery
```

Exception:

```bash
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/usr/miniconda3/envs/fastdup/lib/python3.9/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

Qt is installed. Also tried reinstalling it but it didn't seem to work.

**Comment by dbickson** [comment]
Hi @mesllo we have never seen this one either.
Can you please send the full stack trace so we will see which call resulted? I suspect it may be related to matplotlib.
I found this solution on the web:
https://github.com/NVlabs/instant-ngp/discussions/300
Let us know if this helps?

**Comment by mesllo** [comment]
Can confirm that the solution that you linked worked. Basically, you have to do the following prior to running the code:

`export QT_QPA_PLATFORM=offscreen`

There doesn't seem to be a full stack trace for the issue but this is my full output:

```bash
FastDup Software, (C) copyright 2022 Dr. Amir Alush and Dr. Danny Bickson.
Warning: error writing config file: [Errno 2] No such file or directory: 'runs/dataset/out' to file runs/out/config.json
Going to loop over dir datasets/dataset
Found total 184 images to run on
[■■■■■■■                                           ] 14%[■■■■■■■■■■■■■■                                    ] 28%[■■■■■■■■■■■■■■■■■■■■■                             ] 41%[■■■■■■■■■■■■■■■■■■■■■■■■■■■■                      ] 55%[■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■                ] 68%[■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■         ] 82%[■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■  ] 96%[■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■] 100Found total 184 images to run ons
41) Finished write_index() NN model
Stored nn model index file runs/dataset/out/nnf.index
Total time took 6062 ms
Found a total of 52 fully identical images (d>0.990), which are 4.71 %
Found a total of 339 nearly identical images(d>0.980), which are 30.71 %
Found a total of 821 above threshold images (d>0.900), which are 74.37 %
Found a total of 46 outlier images         (d<0.050), which are 4.17 %
Min distance found 0.751 max distance 0.995
1671195388 : INFO:     (add_vertices:460): Num vertices for group 0: 184
1671195388 : INFO:     (commit_edge_buffer:609): In commit edge buffer (0,0)
1671195388 : INFO:     (commit_edge_buffer:680): Shuffling edges ...
1671195388 : INFO:     (commit_edge_buffer:688): Done shuffling edges in 0.015967 secs
1671195388 : INFO:     (commit_edge_buffer:692): Aggregating unique vertices...
1671195388 : INFO:     (commit_edge_buffer:705): Done aggregating unique vertex in 0.001447 secs
1671195388 : INFO:     (commit_edge_buffer:713): Combine vertex data
1671195388 : INFO:     (commit_edge_buffer:779): Done phase 2 in 0.000982 secs
1671195388 : INFO:     (commit_edge_buffer:787): Rename id columns 
1671195388 : INFO:     (commit_edge_buffer:890): Done in 0.012565 secs
1671195388 : INFO:     (commit_edge_buffer:892): Finish committing edge in 0.031603 secs
1671195388 : INFO:     (add_edges:584): Num vertices for group 0: 184
Num vertices for group 0: 184
Num edges 0 -> 0: 631
1671195389 : PROGRESS: (_p:516): +-----------------------------+
1671195389 : PROGRESS: (_p:516): | Number of components merged |
1671195389 : PROGRESS: (_p:516): +-----------------------------+
1671195389 : PROGRESS: (_p:516): | 349                         |
1671195389 : PROGRESS: (_p:516): | 0                           |
1671195389 : PROGRESS: (_p:516): +-----------------------------+
1671195389 : PROGRESS: (triple_apply_pagerank:69): Counting out degree
1671195389 : PROGRESS: (triple_apply_pagerank:78): Done counting out degree
1671195389 : PROGRESS: (_p:516): +-----------+-----------------------+
1671195389 : PROGRESS: (_p:516): | Iteration | L1 change in pagerank |
1671195389 : PROGRESS: (_p:516): +-----------+-----------------------+
1671195389 : PROGRESS: (_p:516): | 1         | 90.8083               |
1671195389 : PROGRESS: (_p:516): | 2         | 23.6306               |
1671195389 : PROGRESS: (_p:516): | 3         | 10.6009               |
1671195389 : PROGRESS: (_p:516): | 4         | 6.24823               |
1671195389 : PROGRESS: (_p:516): | 5         | 4.12893               |
1671195389 : PROGRESS: (_p:516): | 6         | 2.93532               |
1671195389 : PROGRESS: (_p:516): | 7         | 2.17884               |
1671195389 : PROGRESS: (_p:516): | 8         | 1.65517               |
1671195389 : PROGRESS: (_p:516): | 9         | 1.27083               |
1671195389 : PROGRESS: (_p:516): | 10        | 0.986649              |
1671195389 : PROGRESS: (_p:516): | 11        | 0.772858              |
1671195389 : PROGRESS: (_p:516): | 12        | 0.6108                |
1671195389 : PROGRESS: (_p:516): | 13        | 0.486701              |
1671195389 : PROGRESS: (_p:516): | 14        | 0.390799              |
1671195389 : PROGRESS: (_p:516): | 15        | 0.31597               |
1671195389 : PROGRESS: (_p:516): | 16        | 0.257053              |
1671195389 : PROGRESS: (_p:516): | 17        | 0.210263              |
1671195389 : PROGRESS: (_p:516): | 18        | 0.172806              |
1671195389 : PROGRESS: (_p:516): | 19        | 0.142599              |
1671195389 : PROGRESS: (_p:516): | 20        | 0.118084              |
1671195389 : PROGRESS: (_p:516): +-----------+-----------------------+
Wrote total of 184 components 
runs/dataset/out/similarity.csv
Failed to read config file runs/dataset/out/config.json
100%|███████████████████| 20/20 [00:01<00:00, 12.85it/s]
Stored similarity visual view in  runs/dataset/similarity.html
Failed to read config file runs/dataset/out/config.json
read outliers 46
100%|████████████████| 17/17 [00:00<00:00, 18520.30it/s]
Stored outliers visual view in  runs/dataset/outliers.html
Failed to read config file runs/dataset/config.json
100%|███████████████████| 14/14 [00:22<00:00,  1.60s/it]
Finished OK. Components are stored as image files runs/dataset/components_[index].jpg
Stored components visual view in  runs/dataset/components.html
Execution time in seconds 22.8
QObject::moveToThread: Current thread (0x2df1b50) is not the object's thread (0xb6b8700).
Cannot move to target thread (0x2df1b50)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/usr/miniconda3/envs/fastdup/lib/python3.9/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, minimal, minimalegl, offscreen, vnc, webgl.
```

**Comment by dbickson** [comment]
Should be fixed in 0.180 by using open-cv-headless instead
